FSW/src/util/commandHandling.cpp

```diff
@@ -430,13 +440,28 @@ void executeCommand(int command) {
  *  None
  */
 void printInfo() {
-    Serial.println("===== Status Report =====");
+    Serial.println("========== Status Report ==========");
     Serial.print("Uptime (ms): ");
     Serial.println(millis());
     Serial.print("Mode: ");
-    Serial.println(scienceMode.getMode());
+    switch (scienceMode.getMode()) {
+        case SAFE_MODE: Serial.println("Safe");
+            break;
+        case STANDBY_MODE: Serial.println("Standby");
+            break;
+        case SUNSET_MODE: Serial.println("Sunset");
+            break;
+        case PRE_SUNRISE_MODE: Serial.println("Watching for Sunrise");
+            break;
+        case SUNRISE_MODE: Serial.println("Sunrise");
+            break;
+        default: Serial.println("Mode Not Recognized!");
```

Comment:
If I'm the test operator and I see this, I'd be pretty scared. Maybe switch to save mode and also print that?

Reply:
`setMode()` will already default to safe mode if the mode isn't recognized, so I just put that in for the sake of having a default case really